airflow/config_templates/config.yml

```diff
@@ -477,22 +477,22 @@ core:
       type: string
       default: "0o077"
       example: ~
-    dataset_manager_class:
-      description: Class to use as dataset manager.
-      version_added: 2.4.0
+    asset_manager_class:
+      description: Class to use as asset manager.
```

Comment:
Should we mention this is renamed? (Is there a `version_changed` for this?)

Reply:
This is mentioned in `newsfragments/41348.significant.rst`. Is there any where you want us to mention it again?

Reply:
We just decide this would be a bad idea if we add it to description